tests/test_western/test_tremolo.py

```diff
@@ -0,0 +1,41 @@
+from neoscore.core.point import ORIGIN, ZERO
+from neoscore.core.units import Mm
+from neoscore.western.chordrest import Chordrest
+from neoscore.western.clef import Clef
+from neoscore.western.staff import Staff
+from neoscore.western.tremolo import Tremolo
+
+from ..helpers import AppTest
+
+
+class TestTremolo(AppTest):
+    def setUp(self):
+        super().setUp()
+        self.staff = Staff(ORIGIN, None, Mm(200))
+        clef = Clef(ZERO, self.staff, "treble")
+        self.cr1 = Chordrest(Mm(10), self.staff, ["c'"], (1, 2))
+        self.cr2 = Chordrest(Mm(20), self.staff, ["d"], (1, 2))
+
+    def test_init_with_int_indication_args(self):
+        tremolo = Tremolo(ORIGIN, self.staff, 3)
+        assert tremolo.parent == self.staff
+        assert tremolo.glyph_name == "tremolo3"
+        assert tremolo.pos.x == ZERO
+        assert tremolo.pos.y == ZERO
+
+    def test_init_with_smufl_indication_args(self):
+        tremolo = Tremolo(ORIGIN, self.staff, "pendereckiTremolo")
+        assert tremolo.parent == self.staff
+        assert tremolo.glyph_name == "pendereckiTremolo"
+        assert tremolo.pos.x == ZERO
+        assert tremolo.pos.y == ZERO
+
+    def test_ini_build_with_for_chordrest_function(self):
+        tremolo = Tremolo.for_chordrest(self.cr2, 3)
+        assert tremolo.parent == self.cr2
+        assert tremolo.glyph_name == "tremolo3"
+        assert tremolo.pos == tremolo.tremolo_position
+        # todo - this last test shows and error that doesnlt make sense, as the values do match
+        # assert tremolo.pos == Point(
+        #     self.staff.unit(0.067), self.staff.unit(2.996)
+        # )
```

Comment:
When I run it the error I get is for clearly different values:

```
E       AssertionError: assert Point(x=Staff...ffUnit(2.895)) == Point(x=Staff...ffUnit(2.996))
E
E         Differing attributes:
E         ['x', 'y']
E
E         Drill down into differing attribute x:
E           x: StaffUnit(0.291) != StaffUnit(0.067)
E         ...
E
E         ...Full output truncated (4 lines hidden), use '-vv' to show
```

but the error you were seeing is probably caused by attempting a strict `==` between points - you can get away with this sometimes, but it's much more reliable to use `assert_almost_equal`, provided in test/helpers.py.